File: fix.js
const fs = require("fs");
const files = [
  "c:/Users/stuti/OneDrive/Desktop/FunWithArts/funwithart-main/studio.html",
  "c:/Users/stuti/OneDrive/Desktop/FunWithArts/funwithart-main/wishlist.html",
];
const close = "</" + String.fromCharCode(100, 105, 118) + ">";
const open = "<" + String.fromCharCode(100, 105, 118);
for (const p of files) {
  let t = fs.readFileSync(p, "utf8");
  t = t.replace(/<\/motion>/g, close);
  t = t.replace(/<motion /g, open + " ");
  t = t.replace(/<motion>/g, open + ">");
  fs.writeFileSync(p, t);
}

File: _fix.py
import pathlib
p = pathlib.Path(r"c:\Users\stuti\OneDrive\Desktop\FunWithArts\funwithart-main\login.html")
t = p.read_text(encoding="utf-8")
bad_o = "<" + "motion" + ">"
bad_c = "</" + "motion" + ">"
good_o = "<" + "div" + ">"
good_c = "</" + "motion" + ">"
good_c = "</" + "motion" + ">"
